compiler/codegen/cg-expr.cpp

```diff
@@ -2166,11 +2166,8 @@ GenRet codegenTernary(GenRet cond, GenRet ifTrue, GenRet ifFalse)
 
     func->getBasicBlockList().push_back(blockEnd);
     info->irBuilder->SetInsertPoint(blockEnd);
-#if HAVE_LLVM_VER >= 130
-    ret.val = info->irBuilder->CreateLoad(tmp->getType()->getPointerElementType(), tmp);
-#else
-    ret.val = info->irBuilder->CreateLoad(tmp);
-#endif
+    ret.val = info->irBuilder->CreateLoad(ret.chplType->symbol->getLLVMStructureType(), tmp);
```

Comment:
I'm assuming providing type was optional but allowed in LLVM 11 and 12, since we are now doing that for those versions where we weren't before.

Reply:
Right, the type was optional and if you didn't provide a type it used to the getPointerElementType() thing inside of CreateLoad.